src/filters/utils.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module pastefromoffice/filters/utils
+ */
+
+/* globals DOMParser */
+
+import DomConverter from '@ckeditor/ckeditor5-engine/src/view/domconverter';
+import { NBSP_FILLER } from '@ckeditor/ckeditor5-engine/src/view/filler';
+
+const domParser = new DOMParser();
+const domConverter = new DomConverter( { blockFiller: NBSP_FILLER } );
+
+/**
+ * Parses provided HTML extracting contents of `body` and `style` tags.
+ *
+ * @param {String} htmlString HTML string to be parsed.
+ * @returns {Object} result
+ * @returns {module:engine/view/documentfragment~DocumentFragment} result.body Parsed body
+ * content as a traversable structure.
+ * @returns {String} result.bodyString Entire body content as a string.
+ * @returns {Array.<CSSStyleSheet>} result.styles Array of native `CSSStyleSheet` objects, each representing
```

Comment:
Did you try to build docs? JSDoc may complain about this. We'd need to add it to the known types there.

Reply:
It seems to be working fine. No errors during docs build.
![image](https://user-images.githubusercontent.com/1061942/45950480-45e57a00-c000-11e8-82fa-d765daf6ddc5.png)